### ISSUE TITLE
Dir example `Program exited early with error: (UnableToCreateFirstDir (DirError WriteZero))`

### ISSUE THREAD

**Issue by lukewilliamboswell** [roc-lang/basic-cli#193]
Dir example is behaving strangely in CI. Adding an exception to skip it for now, this issue is to track the problem.

Needs further investigation.

```
❯ rm -rf dirExampleE/ && valgrind examples/dir
==126467== Memcheck, a memory error detector
==126467== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==126467== Using Valgrind-3.22.0 and LibVEX; rerun with -h for copyright info
==126467== Command: examples/dir
==126467== 
==126467== Conditional jump or move depends on uninitialised value(s)
==126467==    at 0x1147E10: ??? (in /home/username/gitrepos/basic-cli/examples/dir)
==126467==    by 0x114A150: ??? (in /home/username/gitrepos/basic-cli/examples/dir)
==126467==    by 0x1FFEFEF28F: ???
==126467==    by 0x4A600B7: ???
==126467==    by 0x4034AAF: ??? (in /nix/store/ddwyrxif62r8n6xclvskjyy6szdhvj60-glibc-2.39-5/lib/ld-linux-x86-64.so.2)
==126467==    by 0x114B9D2: ??? (in /home/username/gitrepos/basic-cli/examples/dir)
==126467==    by 0x1FFEFEF447: ???
==126467==    by 0x1FFEFEF2BF: ???
==126467==    by 0x4: ???
==126467==    by 0x4A694CF: ???
==126467==    by 0x8: ???
==126467==    by 0x1145E32: ??? (in /home/username/gitrepos/basic-cli/examples/dir)
==126467== 
==126467== Conditional jump or move depends on uninitialised value(s)
==126467==    at 0x1142FCD: ??? (in /home/username/gitrepos/basic-cli/examples/dir)
==126467==    by 0x11444A0: ??? (in /home/username/gitrepos/basic-cli/examples/dir)
==126467==    by 0x1144359: ??? (in /home/username/gitrepos/basic-cli/examples/dir)
==126467== 
Program exited early with error: (UnableToCreateFirstDir (DirError WriteZero))
==126467== 
==126467== HEAP SUMMARY:
==126467==     in use at exit: 0 bytes in 0 blocks
==126467==   total heap usage: 16 allocs, 16 frees, 2,415 bytes allocated
==126467== 
==126467== All heap blocks were freed -- no leaks are possible
==126467== 
==126467== Use --track-origins=yes to see where uninitialised values come from
==126467== For lists of detected and suppressed errors, rerun with: -s
==126467== ERROR SUMMARY: 2 errors from 2 contexts (suppressed: 0 from 0)
```

**Comment by Anton-4** [comment]
Can this be closed?

**Comment by lukewilliamboswell** [comment]
Fixed by #188
